CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contributing
+
+If you would like to make changes to this repo, please:
+
+1. Make a GitHub Pull Request with the suggested change
+2. Post the Pull Request in the Slack room: #Frontend
+3. Wait 24 hours
```

Comment:
I would extend it a bit more, maybe three days?

I would also mention to re-share it just before merging